sdk/armcore/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,105 @@
+## Guide for migrating to `sdk/**/arm**` from `services/**/mgmt/**`
+
+This document is intended for users that are familiar with an older version of the Azure SDK For Go for management modules (`services/**/mgmt/**`) and wish to migrate their application to the next version of Azure resource management libraries (`sdk/**/arm**`)
```

Comment:
Will the new version policy impact the migration?

Reply:
Sorry but I did not get the point. Could you be more specific?

Reply:
I mean we changed from multi-api version to single api version, any behaviour change from user's prospective?